src/main/java/ch/ivyteam/ivy/maven/DeployToEngineMojo.java

```diff
@@ -284,6 +284,13 @@ else if (DeployMethod.HTTP.equals(deployMethod))
   private void deployToDirectory(File resolvedOptionsFile) throws MojoExecutionException
   {
     File deployDir = getDeployDirectory();
+    if (deployEngineDirectory == null)

```

Comment:
Do I really need to set "deployEngineDirectory" always? Or how does this work?

Reply:
This log covers a special case; when you have an <engineDirectory> configured (which as we said is a legacy approach) and want to manually deploy something to it.

before it would find the deploy directory via a maven property but because the engine can now be copied to the target this property doesn't get resolved anymore. So this should really only happen if you **force the copying via COPY_FROM_TEMPLATE** and then want to deploy to this engine in an individual deploy execution.

Reguel meant that it's normal if you have to declare the <deployEngineDirectory> (is what I heard from Lukas)